.travis.yml

```diff
@@ -1,47 +1,46 @@
 language: go
-
 sudo: required
-
 go:
-  - 1.9.x
-  - master
-
+- 1.9.x
```

Comment:
This file got reformatted by the travis CLI tool. I'm inclined to leave the new formatting so that in the future if we use the tool it doesn't reformat it again.